website/docs/index.html.markdown

```diff
@@ -88,7 +88,7 @@ The following arguments are supported:
 * `registry_config_path` - (Optional) The path to the registry config file. Defaults to `HELM_REGISTRY_CONFIG` env if it is set, otherwise uses the default path set by helm.
 * `repository_config_path` - (Optional) The path to the file containing repository names and URLs. Defaults to `HELM_REPOSITORY_CONFIG` env if it is set, otherwise uses the default path set by helm.
 * `repository_cache` - (Optional) The path to the file containing cached repository indexes. Defaults to `HELM_REPOSITORY_CACHE` env if it is set, otherwise uses the default path set by helm.
-* `helm_driver` - (Optional) "The backend storage driver. Valid values are: `configmap`, `secret`, `memory`. Defaults to `secret`
+* `helm_driver` - (Optional) "The backend storage driver. Valid values are: `configmap`, `secret`, `memory`, `sql`. Defaults to `secret`
```

Comment:
Maybe we add more detail to the docs on how to configure this? 

What about an additional block for configuring the sql dialect & connection string?

Reply:
It doesn't seem to support anything other than [postgres for now](https://github.com/helm/helm/blob/master/pkg/storage/driver/sql.go#L184) and the way the connection string can be configured is through an environment variable `HELM_DRIVER_SQL_CONNECTION_STRING`. I'll add a note about that in the docs.
I actually had mixed feelings about even adding support for this feature in the provider at this stage, but in the end, I figured it might come in handy for some folks and it might also help out with testing since it's considered a beta.